homeassistant/util/async_.py

```diff
@@ -82,6 +85,14 @@ def check_loop(
     The default advisory message is 'Use `await hass.async_add_executor_job()'
     Set `advise_msg` to an alternate message if the solution differs.
     """
+    # pylint: disable=import-outside-toplevel
+    from homeassistant.core import HomeAssistant, async_get_hass
+    from homeassistant.helpers.frame import (
+        MissingIntegrationFrame,
+        get_integration_frame,
+    )
+    from homeassistant.loader import async_suggest_report_issue
```

Comment:
Should we move this from `util._async_` to `helpers.async_` since we now depend on Home Assistant?

Reply:
That would make sense to me 👍